mlua-sys/src/lua51/compat.rs

```diff
@@ -145,6 +145,12 @@ unsafe fn compat53_pushfuncname(L: *mut lua_State, ar: *mut lua_Debug) {
 // lua ported functions
 //
 
+#[inline(always)]
+pub unsafe fn lua_error(L: *mut lua_State) -> ! {
+    lua_error_(L);
+    unreachable!()
+}
```

Comment:
It's not longer needed, see https://github.com/khvzak/mlua/commit/59974d7bdeaf7c1581198128e7ec275220664db0